lms/djangoapps/bulk_email/tasks.py

```diff
@@ -491,9 +491,14 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
     course_title = global_email_context['course_title']
     course_language = global_email_context['course_language']
 
-    # use the email from address in the CourseEmail, if it is present, otherwise compute it
-    from_addr = course_email.from_addr if course_email.from_addr else \
```

Comment:
What is this `CourseEmail` address it refers to?

Reply:
If I'm not mistaken that's an entry in the db for the course email that might exists or not